hostpath-provisioner.go

```diff
@@ -94,12 +94,21 @@ func isCorrectNode(annotations map[string]string, nodeName string) bool {
 	return false
 }
 
+func (p *hostPathProvisioner) ShouldProvision(pvc *v1.PersistentVolumeClaim) bool {
```

Comment:
I would've thought the qualifer would need declared for this.  If this works though that's great.

Reply:
I am not quite sure what you mean by qualifier, but yes this works.

Reply:
I was wrong, don't worry about it :)  I was wondering if it needed a declaration or somethign specifically ton indicate it was part of the qualifier interface, but it looks like that all bubbles up through the volume package.